sensor_msgs/Image image
---
BoundingBoxes yolo_result
---
bool error
